### PR TITLE
Add Project Dependencies

### PR DESCRIPTION
**Pull request by sopermaf**
Really like the idea of the project!

Couple of ideas:
1. added `poetry` to help with managing requirements
2. added some dev dependencies for auto linting (black, isort, flake8)
3. moved the existing python modules into a single `ai-speed-camera` and updated README for the samples files

### CODE REVIEW

**Comment by jthomas** [comment]
Thank you for opening this! I will do a review and merge it in a few days. Feel free to open any others.

**Comment by sopermaf** [comment]
> Thank you for opening this! I will do a review and merge it in a few days. Feel free to open any others.

No prob! Just did it for fun as I was inspired by the idea! Saw it over on Twitter and wanted to check it out,

Cool idea btw! :100:  I want to get it running on my own road now!

I won't add any changes within this PR (to keep it smalli) so if you feel like accepting the changes go ahead :smile:

File: README.md [hunk 1]
```diff
@@ -49,15 +58,18 @@ https://videointelligence.googleapis.com/v1/projects/<X>/locations/<REGION>/oper
...
+curl -X GET \
 -H "Authorization: Bearer "$(gcloud auth application-default print-access-token) \
 https://videointelligence.googleapis.com/v1/projects/<X>/locations/<REGION>/operations/<Y> > results.json
 ```
 
 ### Run Python Script To Annotate Video
 
-```
-python3 init.py --video input.mp4 --output dest.mp4 --annotations results.json --distance 32
+```shell
```
Comment (sopermaf) [inline README.md:68]:
```suggestion
```
```

File: ai_speed_camera/annotations_processor.py [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
 
 seconds = re.compile("\d+(\.\d+)?")
 
 line_midpoint = lambda start, end: (start + end) / 2
 
-bb_centroid = lambda bb: (line_midpoint(bb['left'], bb['right']), line_midpoint(bb['top'], bb['bottom']))
+bb_centroid = lambda bb: (
+    line_midpoint(bb["left"], bb["right"]),
+    line_midpoint(bb["top"], bb["bottom"]),
+)
```
Comment (sopermaf) [inline ai_speed_camera/annotations_processor.py:16]: Just black working it's magic. Flake8 shouts about this but I figured better leave for another PR

File: README.md (hunk 1 above)
Comment (sopermaf) [reply README.md:68]: Adds nothing from what I've seen

File: ai_speed_camera/annotations_processor.py (hunk 2 above)
Comment (sopermaf) [reply ai_speed_camera/annotations_processor.py:16]: Code changes are just `isort` and `black` apart from removing unused `sys` import and moving logging setup, so seem to be safe to use.

I ran with the sample data without issue so I think that's all required to confirm?

**Review by jthomas — APPROVED** [review]
